### PR TITLE
Apache 2.0 License

### LINKED ISSUE #53: Issue: Remove Testing Section from README
### Issue
Outdated testing section exists in the README.

### Fix
It should just be removed from the README.

### PR DESCRIPTION
**Pull request by wavious-wpatty**
Fixes:
  - Fixes #53

Features:
  - Apache-2.0 License (#57)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,8 @@ See [wav-lpddr-hw](https://github.com/waviousllc/wav-lpddr-hw) for more details
...
-* The WAV-LPDDR-SW application code (located under `app` directory) and other
-  portions not explicitly licensed otherwise, are licensed under the
-  GNU GENERAL PUBLIC LICENSE -- see `COPYING` file in this directory for
-  details.
-
-* `libwddr` and the other WAV-LPDDR-SW libraries are licensed under the
-  GNU LESSER GENERAL PUBLIC LICENSE -- see `COPYING.LESSER` file in this
-  directory for details.
+This project is licensed as Apache-2.0. Previous releases used GPL-3.0 and
+LGPL-3.0.
```
Comment (wavious-jbasista) [inline README.md:9]: Do we even have to acknowledge what it was previously under?  Or can we just leave it that it is currently apache?

Comment (wavious-wpatty) [reply README.md:9]: It doesn't have to be there. But I wanted something there incase someone ever came across the non-apache version.

Comment (wavious-jbasista) [reply README.md:9]: Ok, works for me, just wanted to make sure it didn't make it more confusing, but should be ok.

**Review by wavious-jbasista — APPROVED** [review]
Everything looks good, don't see any trace of GPL or LGPL in files.

**Pull request merged**